src/parser/jobs/pld/modules/OGCDDowntime.tsx

```diff
@@ -6,5 +6,6 @@ export default class OGCDDowntime extends CooldownDowntime {
 	trackedCds = [
 		ACTIONS.SPIRITS_WITHIN.id,
 		ACTIONS.CIRCLE_OF_SCORN.id,
+		ACTIONS.INTERVENE.id,
```

Comment:
Is intervene something you actually want to use on CD?

Reply:
Nope. But idk how charges are being implemented. The rule of thumb is: "Use 1 charge if you gain a GCD or an Intervene. Otherwise dump 2 into every FoF."

Reply:
Fukkin. Was confusing intervene with intervention, hence the comment.

Works for now, can test how it jives with real stuff.